app/models/contact/Contact.php

```diff
@@ -302,7 +302,7 @@ function getAlbum()
         $uri = str_replace(
```

Comment:
You can simply remove the getAlbum() methode, it's not called anywhere anymore

Reply:
Ok, I will.